app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -100,7 +100,8 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             stopSelf();
             return START_NOT_STICKY;
         }
-        if (CollectionServiceStarter.isBTWixel(getApplicationContext()) || CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
+        if (CollectionServiceStarter.isBTWixel(getApplicationContext()) || CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())
```

Comment:
I'm seeing this block of comparisons a lot.  Maybe there should be a CollectionServiceStarter.isWixel method to clean it and some of the other big ifs up?  Maybe it should just be that isBTWixel and isWifiWixel return true for the combined case.  It might not be in the scope of this PR, but this is starting to smell like we need some better abstractions around devices to make this easier.